spec/zenaton/services/properties_spec.rb

```diff
@@ -95,7 +95,7 @@
       let(:object) { OpenStruct.new(a: 1) }
 
       it 'returns the struct instance variables table' do
-        expect(result).to eq('t' => { 'a' => 1 })
```

Comment:
Is this a breaking change? Why we have to change our test here?

Reply:
To be honest this one puzzled me for a while. The value of under the `'t'` key comes straight from `ostruct` itself. Looking at https://github.com/ruby/ruby/blame/930e637c0e2ffd6fcb46205d473b6f4c05607471/lib/ostruct.rb#L95 this was always a symbol-keyed hash.

I believe the reason for the change is that there was a `JSON.parse(object.to_json)` inside the old `from_complex_type` method. https://github.com/zenaton/zenaton-ruby/pull/78/files#diff-23c60c8b7e6b27a6e1526cfb0f17422fL109 and this is now gone.

I'm adding a test case to make sure the decoding can handle both forms.